nixio/block.py

```diff
@@ -358,7 +349,7 @@ def create_data_frame(self, name="", type_="", col_dict=None,
         if col_dict is not None:
             for nam, dt in col_dict.items():
                 if isclass(dt):
-                    if any(issubclass(dt, st) for st in string_types) \
+                    if any(issubclass(dt, st) for st in str) \
```

Comment:
A lot of the failures seems to be related to these changes, where an iterable `string_types` was replaced with a plain type `str` and the iteration wasn't dropped.
This should change to:
```
if issubclass(dt, str):
```
(or `isinstaince()`) since we only have one string type now.

Reply:
Hi, I don't mind

Reply:
If I'm not failing I'm not trying hard enough. Here the codebase was a tiny bit confusing because in some places it is know that `instance()` will work on a tuple and in others places there was this `any()` construct.